scene/resources/curve.h

```diff
@@ -280,6 +280,7 @@ class Curve3D : public Resource {
 	Vector3 _sample_baked(Interval p_interval, bool p_cubic) const;
 	real_t _sample_baked_tilt(Interval p_interval) const;
 	Basis _sample_posture(Interval p_interval, bool p_apply_tilt = false) const;
+	Basis _compose_posture_baked(int p_baked_idx) const;
```

Comment:
```suggestion
	Basis _compose_posture(int p_idx) const;
```

For consistency with other methods.

Reply:
done